app/views/application/_user-reminders.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render "email-confirmation-reminder" %>
+
+<%= render "insecure-password-reminder" %>
```

Comment:
Are all these partials necessary? The two partials this one includes are only 8 and 10 lines long each, would it make reading and understanding the code easier if they were both combined into this file?